Symp-78 | Jul20 copy changes

taraepp opened the pull request:
Ticket: https://freshworks.atlassian.net/browse/SYMP-78
This should encompass all current document changes, with the exception of updating icons

ninosamson commented on apps/web/src/components/Home.tsx:
```diff
@@ -12,11 +12,12 @@ const Home = () => {
+          <Trans t={t} i18nKey={'Last updated'}>
             <div>
-              <img src={banner} alt='government of british columbia' className='w-full' />
+              Last updated: <b>July 7, 2022</b>
```
last updated: July 28, 2022

dbayly-freshworks reviewed (approved):
Great work. I only have a few comments: 
- Missing Telephone link on moderate sore throat description
- Missing telephone link on severe diarrhea description
- We should get clarity on the list for the vulnerability questions 
- NIT space missing between healthink BC link and word `about` in the headache question

taraepp commented:
> * Missing Telephone link on moderate sore throat description

Addressed this one

> * Missing telephone link on severe diarrhea description

Did not address this one (it has two 8-1-1's, 1 in title, 1 in description. 811 in description is a link, but not title- title would need minor refactor to allow formatting)

> * We should get clarity on the list for the vulnerability questions

Did not address this one, I see now that I was copying from a list from Jun 30!

> * NIT space missing between healthink BC link and word `about` in the headache question

Addressed this one
